streaming/src/test/scala/io/gearpump/streaming/task/SubscriptionSpec.scala

```diff
@@ -22,7 +22,7 @@ import java.util.Random
 
 import io.gearpump.Message
 import io.gearpump.cluster.UserConfig
-import io.gearpump.partitioner.{Partitioner, HashPartitioner}
+import io.gearpump.streaming.partitioner.{HashPartitioner, Partitioner}
```

Comment:
@manuzhang 

As far as I know, Partitioner and HashPartitioner are not changed. 
MulticastPartitioner is not directly used by end user.